js/packages/components/settings/Home.tsx

```diff
@@ -403,6 +403,21 @@ const HomeBodySettings: React.FC<{ navigation: ComponentProps<typeof Home>['navi
 	items.push({ label: 'Debug', value: 'cimode' })
 	return (
 		<View style={[flex.tiny, padding.horizontal.medium, padding.bottom.small]}>
+			<ButtonSetting
+				name={'AutoPush'}
+				icon='options-outline'
+				iconSize={30}
+				iconColor={colors['background-header']}
+				toggled
+				varToggle={ctx.account?.autoSharePushTokenFlag === true}
+				actionToggle={async () =>
+					await ctx.client?.pushSetAutoShare({
```

Comment:
Can you fix this codefactor error? (either apply the suggestion or ignore the issue)